metrics_ci_integration/test/jenkins/model/jenkins_result_test.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:ci_integration/jenkins/model/jenkins_result.dart';
+import 'package:test/test.dart';
+
+void main() {
+  group('JenkinsResult', () {
+    const failed = JenkinsResult.error(message: 'Error');
+    const success = JenkinsResult.success(message: 'Hooray!');
```

Comment:
Should we also add tests for `result`?

Reply:
Not sure we have to test a property. If we try to cover it with tests we'll either check that Dart's generic works properly or `not-null if not-null provided`.